src/command/client/stats.rs

```diff
@@ -72,32 +68,23 @@ impl Cmd {
         settings: &Settings,
     ) -> Result<()> {
         let context = current_context();
-
-        match self {
-            Self::Day { words } => {
-                let words = if words.is_empty() {
-                    String::from("yesterday")
-                } else {
-                    words.join(" ")
-                };
-
+        let words = if self.period.is_empty() {
+            String::from("all")
+        } else {
+            self.period.join(" ")
+        };
+        let history: Vec<History>;
+        match words.as_str() {
```

Comment:
clippy also recommends using an `if` statement instead of `match` here. My intention of using `match` here is to further extend this to include longer durations like a month, year etc.  Let me know if you would want me to change this to an `if` clause instead,

Reply:
Yeah, I think we can go for an if for now (esp if clippy is going to complain). Converting it back to a match in future is not much work